product-profunctors.cabal

```diff
@@ -23,7 +23,7 @@ source-repository head
 
 library
   default-language: Haskell2010
-  build-depends:   base >= 4.5 && < 5
+  build-depends:   base >= 4.8 && < 5
```

Comment:
But in any case it should be done in a separate commit, and preferably a separate PR!

Reply:
`Data.Void` arrives in `base` at `4.8`. Could go to `4.9` if you wanted.

Reply:
Ah I see. It's interesting you thought of this. It wouldn't have crossed my mind to even check!  (`Data.Void` used to be in the `void` package, so we could *technically* support even further back!)

Reply:
How far back do you really want to go? I would prefer not to have to do conditional dependencies, and `4.8` is GHC 7.10.1.

Reply:
The comment about supporting further back was mostly tongue-in-cheek and for entertainment only! (Nonetheless, I don't think it would require conditional dependencies, rather just unilaterally importing `void`.)

I rescind my previous comment about it needing to be done in a separate PR, now that I understand the rationale.